llvm/tools/llvm-readtapi/llvm-readtapi.cpp

```diff
@@ -79,20 +92,65 @@ static void reportError(Twine Message, int ExitCode = EXIT_FAILURE) {
   exit(ExitCode);
 }
 
+/// Replace extension considering frameworks.
+static void replace_extension(SmallVectorImpl<char> &Path,
+                              const Twine &Extension) {
+  StringRef P(Path.begin(), Path.size());
```

Comment:
I think it would be helpful to add a comment to visualize what is being matched here:
`/System/Library/Frameworks/<NAME>.framework/<NAME>`

Reply:
Will do, I also am considering moving these filesystem helpers into `TextAPI`.